Sources/SwiftUICharts/LabelsView.swift

```diff
@@ -17,14 +19,16 @@ struct LabelsView: View {
     }
 
     var body: some View {
-        HStack(spacing: 0) {
-            ForEach(dataPoints.indexed(), id: \.1.self) { index, bar in
-                if index % self.threshold == 0 {
-                    Text(bar.label)
-                        .multilineTextAlignment(.center)
-                        .foregroundColor(.secondary)
-                        .font(.caption)
-                    Spacer()
+        GeometryReader { geometry in
```

Comment:
Did you check how the layout works inside the scroll view?

Reply:
Do you mean if we do embed multiple graph in a vertical or horizontal scroll view?

I works fine in horizontal and vertical mode. But in that case a graph width has to be define.